lab-procedure.md

```diff
@@ -36,113 +82,84 @@ run_apache:
     - name: apache2
 ```
 
-and apply the state to the master
-```
-salt \*master state.apply apache
-```
-
-Test ensure the standard apache landing page is being served on port 80
-```
-curl localhost | grep "It works!"
-```
-
-Let's also see if we can reach this page from the minions.
-First identify the master's private ip. We can do this by examining the inet listed by ifconfig on eth0. Alternatively, we can use salt's ```network``` module
-
```

Comment:
Why take out reaching the webpage from the minions? I think it's important to understand where web page is being accessed _from_ and it also exposes students to the `network` salt module